File: php/https/post/basic_post.php
<?php

$url = $argv[1];
$user = $argv[2];
$pass = $argv[3];
$postData = $argv[4];

$options = array('http' => array(
	'method' => 'POST',
	'header' => 
		"Authorization: Basic " . base64_encode("$user:$pass") . "\r\n" . 
		"Content-Type: application/x-www-form-urlencoded\r\n",
	'content' => $postData
));

$res = file_get_contents($url, false, stream_context_create($options));

echo $res;

?>

File: csharp/https/post/BasicPost.cs
using System;
using System.Net;

class BasicPost
{
	public static void Main(string[] args)
	{
		var url = args[0];
		var user = args[1];
		var pass = args[2];
		var postData = args[3];

		//SSLØ–¾‘‚ğ–³‹
		ServicePointManager.ServerCertificateValidationCallback = 
			(sender, cert, chain, errors) => true;

		WebClient wc = new WebClient();

		//Basic”FØ
		wc.Credentials = new NetworkCredential(user, pass);

		//Post
		var res = wc.UploadString(url, "POST", postData);

		Console.Write(res);
	}
}
